__tests__/__integration__/imperative/__tests__/__integration__/cli/config/__scripts__/delete_configs.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+rm $1
```

Comment:
What do you think about moving small scripts to the Testing Environment "APIs"?

No changes requested.
I like the phrase "if it isn't broken, don't try to fix it" 😋

Reply:
It would be nice to have it in the test environment APIs. We can look into that later, when we have more capacity.